### PR TITLE
Remove confusing start list value

### PR DESCRIPTION
**Pull request by bahiirwa**
Works as a pun but Onscreen readers & visually is confusing.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by bahiirwa** [comment]
@nylen Thanks for highlighting this. [GNU project](https://www.gnu.org/philosophy/free-sw.html) I had not seen it.

The change might not consequential in any way. I tried voice over (accessibility) and it works fine. Its read as item one. However, visually, I didn't get the pun until i over thought it.

The way GNU project styles it make it easy to take in. Let me know if this is plausible. Otherwise, we should close PR. Thanks for the time

**Comment by nylen** [comment]
> The way GNU project styles it make it easy to take in.

Sure, this would be fine too.

I'll take a look at the failing test when I'm back at the computer. Probably just a random failure rather than a problem with this PR.

**Comment by johnalarcon** [comment]
Note that the list numbering can _seem_ inconsistent in some situations. For example, WordPress lists the freedoms as 0-3, same as noted above, but then there are public-facing pages with the following. While it makes sense once you know about it, to an average user, it probably looks like an oversight or error. As long as it's used consistently within CP, it shouldn't be an issue.

image

Side note: In more than a decade with WordPress, I only ever submitted a couple of issues... and this was one of them. [Freedoms 0, 1, 2, 3 out of sync with other public-facing info](https://core.trac.wordpress.org/ticket/44128). 😃

File: src/wp-admin/freedoms.php [hunk 1]
```diff
@@ -50,11 +50,11 @@
 <div class="about-wrap-content">
 	<p class="about-description"><?php printf( __( 'ClassicPress is Free and open source software, built by a distributed community of mostly volunteer developers from around the world. ClassicPress comes with some awesome, worldview-changing rights courtesy of its <a href="%s">license</a>, the GPL.' ), 'https://opensource.org/licenses/gpl-license' ); ?></p>
 
-	<ol start="0">
-		<li><p><?php _e( 'You have the freedom to run the program, for any purpose.' ); ?></p></li>
-		<li><p><?php _e( 'You have access to the source code, the freedom to study how the program works, and the freedom to change it to make it do what you wish.' ); ?></p></li>
-		<li><p><?php _e( 'You have the freedom to redistribute copies of the original program so you can help your neighbor.' ); ?></p></li>
-		<li><p><?php _e( 'You have the freedom to distribute copies of your modified versions to others. By doing this you can give the whole community a chance to benefit from your changes.' ); ?></p></li>
+	<ol>
```
Comment (nylen) [inline src/wp-admin/freedoms.php]: > As long as it's used consistently within CP, it shouldn't be an issue.

In this spirit, and to match the styling from the GNU project, let's change this to a `ul` element.  Currently the list says "1. You have the freedom to run the program, for any purpose (freedom 0)."

**Pull request closed without merging**